Shared/Samples/Show popup/ShowPopupView.swift

```diff
@@ -0,0 +1,74 @@
+// Copyright 2022 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+
+struct ShowPopupView: View {
+    /// A map of reported incidents in San Francisco.
+    @State private var map = Map(url: URL(string: "https://arcgisruntime.maps.arcgis.com/home/item.html?id=fb788308ea2e4d8682b9c05ef641f273")!)!
```

Comment:
In the samples we prefer using portal item over URLs

- No need to unwrap URL
- Make it easier to search for/pattern match all the portal item IDs we use in the samples
- Code looks shorter

```swift
    @State private var map = Map(
        item: PortalItem(
            portal: .arcGISOnline(connection: .anonymous),
            id: .incidentsInSanFrancisco
        )
    )

// at the bottom
private extension PortalItem.ID {
    /// The ID used in the "Incidents in San Francisco" portal item.
    static var incidentsInSanFrancisco: Self { Self("fb788308ea2e4d8682b9c05ef641f273")! }
}
```